fx2ait/fx2ait/csrc/AITModelImpl.cpp

```diff
@@ -17,7 +23,9 @@
 namespace torch::aitemplate {
 
 AITemplatePyTorchCachingAllocator::AITemplatePyTorchCachingAllocator() {
+  #ifndef __HIP_PLATFORM_HCC__
```

Comment:
@ipiszy @chenyang78 I'm just highlighting some change, hopefully to make the review a little easier on your side